server/validate.js

```diff
@@ -30,3 +30,15 @@ exports.validateLogin = [
     next();
   }
 ];
+
+exports.validateUserTypeQuery = [
```

Comment:
I don't think we'll need this anymore though but you can leave it in for now 😄

Reply:
Ya I will remove it and replace it with other validation.